cdap-docs/developers-manual/source/data-exploration/index.rst

```diff
@@ -22,10 +22,12 @@ Data Exploration
 
 
 This section covers how you can explore data in CDAP through the use of ad-hoc SQL-like queries.
-Queries can be run over streams and certain types of datasets.
+Queries can be run over streams and certain types of datasets. We refer to this as *CDAP Explore*,
+or *Explore* for short. In this documentation of *Explore*, we use the word *dataset* exclusively 
+here, to refer to either a stream or an *Explore*\ -enabled dataset.
```

Comment:
I don't think this is correct. A dataset is a dataset and does not refer to a stream.